libs/design-system/react-scanner.config.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Configuration file for react-scanner: https://github.com/moroshko/react-scanner
+ *
+ * Used to assess usage of Mantine and Design System components in web.
+ *
+ * To use:
+ * 1. Run `pnpm audit-components`
+ * 2. Check `OUTPUT_PATH` for your scan results!
+ */
+
+/** the path of the scan output */
+const OUTPUT_PATH = './src/component-audit/component-scans';
+const OUTPUT_FILE_NAME = 'scan';
+const OUTPUT_FILE_EXTENSION = 'json';
+
+/**
+ * @param {string} suffix Optional filename suffix
+ * @returns file path for the output file
+ */
+const getOutputFilePath = (suffix) => {
+  return `${OUTPUT_PATH}/${OUTPUT_FILE_NAME}${suffix ?? ''}.${OUTPUT_FILE_EXTENSION}`;
+};
+
+const NOVU_ICON_REGEX = /^Icon(?!Button)[A-Z0-9]{1}[a-zA-Z0-9]+$/;
+const RELATIVE_PATH_REGEX = /^(\.(\.){0,}\/)/;
+const ANTD_ICON_MODULE_NAME = '@ant-design/icons';
+
+module.exports = {
+  /** directory to scan */
+  crawlFrom: '../../apps/web/src/',
+  includeSubComponents: true,
+  /**
+   * Regex for determining which imports to include.
+   * Currently includes: novu, antd, mantine, and local imports
+   *
+   * To see only local imports, replace with: /(\.(\.){0,}\/.*)/gim
+   */
+  importedFrom:
+    /(@novu\/(design-system|shared-web|notification-center)|@mantine\/core|@ant-design)(\/[a-z0-9\-)]+){0,}|(\.(\.){0,}\/.*)/gim,
+  exclude: ['/src/api', '/src/styled-system'],
+  processors: [countComponentsAndPropsProcessor({ minNumInstances: 1 }), groupByNamespaceProcessor],
+  /** file patterns to scan */
+  globs: ['**/!(*.test|*.spec|*.stories).@(js|ts)x'],
```

Comment:
we should also exclude the `*.cy.tsx` files

Reply:
maybe also excluding ones that have suffix `*Page, *Container, *Provider, *Styled, *Sidebar, *Modal`

Reply:
Any reason to exclude those suffixes? They may be files where many other components are brought together and used

Reply:
Sounds good, done!